modules/oe_theme_helper/config/install/responsive_image.styles.oe_theme_main_content.yml

```diff
@@ -6,8 +6,6 @@ dependencies:
     - image.style.oe_theme_medium_no_crop
     - image.style.oe_theme_small_2x_no_crop
     - image.style.oe_theme_small_no_crop
-  theme:
-    - oe_theme
 id: oe_theme_main_content
 label: 'Main content'
 image_style_mappings:
```

Comment:
I leave the comment here but it involves all the moved image styles. We still list them in the theme info file:
![image](https://user-images.githubusercontent.com/153362/97459698-53d2fb00-193c-11eb-9f29-d9a14ad8649d.png)